source/fundamentals/utf8-validation.txt

```diff
@@ -0,0 +1,116 @@
+.. _nodejs-utf-8-validation:
+
+================
+UTF-8 Validation
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to enable or disable the {+driver-short+}'s
+**UTF-8** validation feature. **UTF-8** is a character encoding specification
+that ensures compatibility and consistent presentation across most operating
+systems, applications, and language character sets. By default, the driver
```

Comment:
[comment] It sounds like the default is true/enabled, but that's not 100% clear to me here because the next paragraph starts with "If you enable validation, ..."

Suggest something along the lines of "By default, UTF-8 validation is enabled and the driver checks documents..."

Later on the page does say "By default, the Node.js driver enables UTF-8 validation." but I think that could be clarified up here.

Reply:
I'll include both the statement you suggested and the description. There hasn't been enough context introduced yet by this point in the overview to assume they know what it means to enable UTF-8 validation, so I think having the current info can be helpful. Let me know if that makes sense.

Reply:
It does make sense not to introduce "enabling validation" before providing the context - maybe you could change the next paragraph intro to something like:

"If you use the default setting or explicitly enable validation, the driver ..." 

I don't love my own suggestion, but I do think calling out that "enabled" is the default earlier on would be helpful. 

This isn't a blocking comment, it is ultimately clear on the page and a user doesn't need to know what the default is right away.